COPYRIGHT.txt

```diff
@@ -366,6 +366,11 @@ Copyright: 2011, Khaled Mamou
   2003-2009, Erwin Coumans
 License: BSD-3-clause
 
+Files: ./thirdparty/wslay/
+Comment: wslay
+Copyright: 2011-2015, Tatsuhiro Tsujikawa
+License: MIT
```

Comment:
Since "MIT" is ambiguous (there are different versions of the MIT license), we use "Expat" in the COPYRIGHT.txt file as it's the one listed in the Debian machine-readable copyright spec: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name
(Provided that the wslay license is the same text as the Expat license listed here, but if it's the "usual" MIT it should be.)

Reply:
Yep, it's `Expat`, fixed